__tests__/app.test.js

```diff
@@ -184,29 +187,110 @@ describe("GET - Endpoints", () => {
-                    votes: 0,
-                    created_at: expect.any(String),
-                });
-            });
-    });
-    test('400: Responds with "Bad request" when missing username or body', () => {
+      const newComment = { username: 'butter_bridge', body: 'Great article!' };
       return request(app)
-          .post('/api/articles/1/comments')
-          .send({ username: 'some_name' })
-          .expect(400)
-          .then(({ body: { msg } }) => {
-              expect(msg).toBe('Bad request: missing username or body');
+        .post('/api/articles/1/comments')
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.comment).toEqual({
+            comment_id: expect.any(Number),
+            body: 'Great article!',
+            article_id: 1,
+            author: 'butter_bridge',
+            votes: 0,
+            created_at: expect.any(String),
           });
+        });
+    });
+    test('400: Responds with an error if username/body is missing', () => {
+      const newComment = { body: 'Great article!' };
+      return request(app)
+        .post('/api/articles/1/comments')
+        .send(newComment)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe('Bad request: missing username or body');
+        });
+    });
+    test('400: Responds with an error if article_id is not a number', () => {
+      const newComment = { username: 'butter_bridge', body: 'Great article!' };
+      return request(app)
+        .post('/api/articles/not-a-number/comments')
+        .send(newComment)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe('Invalid article_id');
+        });
+    });
+
+    test('404: Responds with an error if article_id does not exist', () => {
+      const newComment = { username: 'butter_bridge', body: 'Great article!' };
+      return request(app)
+        .post('/api/articles/9999/comments')
+        .send(newComment)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe('Article not found');
+        });
+    });
+
+    test('404: Responds with an error if username does not exist', () => {
+      const newComment = { username: 'nonexistUser', body: 'Great article!' };
+      return request(app)
+        .post('/api/articles/1/comments')
+        .send(newComment)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe('User not found');
+        });
+    });
+
+
+  });
+});
+
+describe('PATCH - Endpoints', () => {
+  describe('PATCH /api/articles/:article_id', () => {
+    test('200: Responds with successfully updates the votes for an article', () => {
+      const updatedVotes = { inc_votes: 5 };
+      return request(app)
+        .patch('/api/articles/1')
+        .send(updatedVotes)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article.votes).toBeGreaterThanOrEqual(0);
```

Comment:
A better expect statement here would be to check the actual value. So if the votes were previously 0, expect them to be 5 👍